src/main/kotlin/no/nav/familie/ef/sak/vedtak/TotrinnskontrollService.kt

```diff
@@ -89,6 +94,20 @@ class TotrinnskontrollService(private val behandlingshistorikkService: Behandlin
             || behandlingStatus == BehandlingStatus.IVERKSETTER_VEDTAK
             || behandlingStatus == BehandlingStatus.OPPRETTET
 
+    fun validerHarGammelGOgKanLagres(saksbehandling: Saksbehandling) {
```

Comment:
Dette ser bra ut, men tenker vi mangler en sjekk. 

Fant ikke notat fra siste statusmötet, tenker vi har følgende tilfeller for åpen behandling (med perioder etter nytt G):
- Har beregnet vedtak med gammelt G, som her blir stoppet
Då må man lagre vedtaket på nytt sånn att nytt G blir brukt

Men så har vi også tilfellet der man har brukt nytt G når man lagrer vedtak, men ikke har revurdert fra første perioden fra nytt G-periode, og der forrige behandling har brukt gammelt G. Her må saksbehandler også lagre vedtaket på nytt fra første perioden etter nytt G beløp? 

Burde denne valideringen også gjøres når man sender til totrinnskontroll? Dvs `SendTilBeslutter` 

Tenker også at sjekken
`saksbehandling.type == BehandlingType.FØRSTEGANGSBEHANDLING || saksbehandling.type == BehandlingType.REVURDERING` egentlige ikke trengs, er kun de 2 som brukes 

For å unngå `?` rundt om så burde man kunne bruke
```kotlin
val tilkjentYtelse = tilkjentYtelseRepository.findByBehandlingId(saksbehandling.id) ?: return
```